### DISCUSSION TITLE
KlipperScreen for different Klipper instances.

### DISCUSSION THREAD

**Discussion by Mate76PL** [KlipperScreen/KlipperScreen#972] [Q&A]
Hey, is there a tutorial anywhere on how to configure KlipperScreen for different Klipper instances. I have 5 instances and I would like KlipperScreen to switch between printers as well. I tried to add according to the documentation Entries: 
# Define printer and name. Name is anything after the first printer word
[printer Rumba]
# Define the moonraker host/port if different from 127.0.0.1 and 7125
moonraker_host: 192.168.88.20
# ports 443 and 7130 will use https/wss
moonraker_port: 7128
# Moonraker API key if this is not connecting from a trusted client IP
moonraker_api_key: False
But I still have default settings in the log: 
'Printer': {
      'moonraker_host': '127.0.0.1',
      'moonraker_port': '7125',
      'moonraker_api_key': '
    }
of course, on the printer connected to port 7125 everything works fine.

**Comment by alfrix** [comment]
when you have issues you need to provide klipperscreen.log 
https://klipperscreen.readthedocs.io/en/latest/Troubleshooting/
